src/docfx/build/Build.cs

```diff
@@ -96,6 +96,12 @@ private static bool ShouldBuildFile(Document itemToBuild, ConcurrentDictionary<D
                 return false;
             }
 
+            if (itemToBuild.ContentType == ContentType.TableOfContents &&
```

Comment:
Can this be done in BuildToc instead of here?

Reply:
I think we should control the build item in one place

---
In reply to: [186925294](https://github.com/dotnet/docfx/pull/2745#discussion_r186925294) [](ancestors = 186925294)